tools/fab/frontend-install.py

```diff
@@ -1,16 +1,35 @@
-#!/usr/bin/python
+#! /opt/stack/bin/python3
 
 from __future__ import print_function
```

Comment:
This can go away.  I general, might be worth seeing which of these imports are still relevant.

Reply:
The linter reports no errors about the other imports. The __future__ import needs to stay in order to support running on python2 long enough to print out an error and exit.